integration-test/src/main/resources/testsuites/clusterostestset.yaml

```diff
@@ -2,8 +2,8 @@ name: "AWS cluster tests on different oses"
 parallel: tests
 threadCount: 4
 parameters:
-  awsRegion: eu-west-1
```

Comment:
I think this file is not used anymore. Instead edit the application.yml. Please correct me if I'm wrong.